Khan/react-render-server
# Instance class change to F4_HIGHMEM

Strangely, it seems according to the node docs that this is F4_1G in the YAML.

Also had to opt yaml files out of prettier.

## Test Plan

1. Deploy this commit SHA (or one including it)
2. Fire a request at it (see our Confluence docs)
3. See that the instance spins up with the higher memory limit in the stackdriver logs
